sdk/include/reactos/version.rc

```diff
@@ -19,7 +19,7 @@
 #define REACTOS_DEFAULT_STR_COMPANY_NAME    "ReactOS Project"
 #define REACTOS_DEFAULT_STR_DESCRIPTION     "ReactOS Core Component"
 #define REACTOS_DEFAULT_STR_INTERNAL_NAME   ""
-#define REACTOS_DEFAULT_STR_LEGAL_COPYRIGHT "Copyright 1996-" COPYRIGHT_YEAR " ReactOS Project"
+#define REACTOS_DEFAULT_STR_LEGAL_COPYRIGHT "(c) 1996-" COPYRIGHT_YEAR " ReactOS Project"
```

Comment:
I found both online. And decided to go with the non-capital one because it was the one that we used before.
But no problem, I can change that. Doesn't matter at all to me which one we use.